lsst-epo/investigations-client
# [C] refactor calculators

JIRA: https://jira.lsstcorp.org/browse/EPO-9131

## What this change does ##

- keep only vertical layout
- limit supernova distance to 3 significant figures
- heavy re-write to remove per-component calculators, try to use generic's to render calculators so that future calculators are simpler to plug in vs. writing a new component

## Notes for reviewers ##

clientside only

Include an indication of how detailed a review you want on a 1-10 scale.
- 3

## Testing ##

- Add both types of calculator to a page
- Verify that the vertical layout looks ok across screen sizes
- Verify that the output of supernova distance is limited to 3 significant figures
- Verify that existing functionality is maintained

## Checklist ##

If any of the following are true, please check them off

## Screenshots (optional) ##
### Before:
### After:
